Gruntfile.js

```diff
@@ -5,6 +5,8 @@ module.exports = function (grunt) {
     express: {
       test: {
         options: {
+          output: "this will never appear",
```

Comment:
This is just a work-around to make sure that server is up & ready before tests start running (waits the time in 'delay' before running next task).
For some reason the 'delay' option is only considered if
A) There is no printed output from the server
AND
B) The string in 'output' is not seen

Using only 'delay' doesn't work since the server prints messages when starting.